### ISSUE TITLE
Connect VCC request for "add repo" command

### ISSUE THREAD

**Issue by momo-the-monster** [vrchat-community/creator-companion#219]
The frontend will expect a websocket message of the following shape:

```json
{
  "messageType": "addRepo",
  "data": { "URL": "someUrl", "headers": { "Accept": "json..... and other optional headers" } }
}
```

The URL could then use the query params for the URL and headers:

```
vcc://vpm/add-repo?url=<URL>&headers[]=<headerName>:<headerValue>&headers[]=<otherHeaderName>:<otherHeaderValue>
```

That's how the headers are handled for the get repo info endpoint which is also using a querystring

**Comment by orels1** [comment]
Now that the backend is done, here's the state of UI


https://user-images.githubusercontent.com/3798928/228706333-5079cd1e-657b-4d65-905f-b65cd9d6328d.mp4

**Comment by momo-the-monster** [comment]
Excellent - looks like this is complete, is there anything else to left to do for this task?

**Comment by orels1** [comment]
I think that is it, I'm just making sure everything is cleaned up and will proceed with a PR

**Comment by orels1** [comment]
This has been released as a part of 2.1.0
